File: test_data/code/python/1266-minimum-time-visiting-all-points.py
import math

class Solution:
    def minTimeToVisitAllPoints(self, points: List[List[int]]) -> int:
        time = 0
        for i in range(1, len(points)):
            dx = abs(points[i][0] - points[i-1][0])
            dy = abs(points[i][1] - points[i-1][1])
            dist = math.sqrt(dx**2 + dy**2)
            time += int(math.ceil(dist))
        return time

File: updated_code/code/java/1266-minimum-time-visiting-all-points.java
package com.example;

import java.util.Arrays;

/**
 * @author wanio
 */
public class Solution {
    public int minTimeToVisitAllPoints(int[][] points) {
        int time = 0;
        for (int i = 0; i < points.length - 1; i++) {
            int dx = Math.abs(points[i + 1][0] - points[i][0]);
            int dy = Math.abs(points[i + 1][1] - points[i][1]);
            // Calculate the minimum time required to visit the points
            time += Math.max(dx, dy);
        }
        return time;
    }
}